docs/topics/caching-mission-resources.adoc

```diff
@@ -0,0 +1,16 @@
+[[about_caching]]
+= Caching Resources
+
+More background and related information on caching can be found here:
+
+ifndef::caching-spring-boot[]
+* link:{link-mission-caching-spring-boot}[{name-mission-caching} - {SpringBoot} Booster]
+endif::caching-spring-boot[]
+
+ifndef::caching-wf-swarm[]
+* link:{link-mission-caching-wf-swarm}[{name-mission-caching} - {WildFlySwarm} Booster]
+endif::caching-wf-swarm[]
+
+ifndef::caching-nodejs[]
+  * link:{link-mission-caching-nodejs}[{name-mission-caching} - {NodeJS} Booster]
+endif::caching-nodejs[]
```

Comment:
What about Vert.x?

Reply:
Missed this one when adding vert.x. Nice catch.